README.md

```diff
@@ -14,53 +12,140 @@ These extensions allow you to test Vue component state and result markup togethe
 #### Create selectors for Vue components
 
 `VueSelector` allows you to select page elements by the component tagName or the nested component tagNames.
-For instance, you can create Vue selectors as follows
+
+Suppose you have the following markup.
 
 ```js
```

Comment:
It will look better with `html` type
```html
<div id="todo-app">
    <todo-input />
    <todo-list>
        <todo-item priority="High">Item 1</todo-item>
        <todo-item priority="Low">Item 2</todo-item>
    </todo-list>   
    <div className="items-count">Items count: <span>{{itemCount}}</span></div>
</div>
<script>
    Vue.component('todo-input', {...});
    Vue.component('todo-list', {...});
    Vue.component('todo-item', {...});
    
    new Vue({ 
        el:   '#todo-app',
        data: {...}
    });
</script>
```